### ISSUE TITLE
Can not pull image cloud-director-named-disk-csi-driver from Harbor 

### ISSUE THREAD

**Issue by tanphan-vn** [vmware/cloud-director-named-disk-csi-driver#46]
### Describe the bug

Can not pull image cloud-director-named-disk-csi-driver from Harbor 

### Reproduction steps

```bash
1.install the cloud-director-named-disk-csi-driver from github for Vmware cloud director kubernetes then can not pull image from Harbor
2. 
3.
...
```

### Expected behavior

Please create image from Harbor

**Comment by goelaashima** [comment]
tanphan-vn

Which branch are you using on your side? Please use the latest version 1.2 from https://github.com/vmware/cloud-director-named-disk-csi-driver/tree/1.2.x branch and that shall pull from Harbor.

**Comment by tanphan-vn** [comment]
Hi @goelaashima yeah, it can pull from this version. Can close this report. But one more thing I would like to ask. Is this applied for VMware cloud director 10.2?

**Comment by arunmk** [comment]
@tanphan-vn the main branch is not released. Hence it has an internal branch name. If you want, you can build the image, push it to a location of your choice, and then change the manifest as well.

CSI doesn't support VMware Cloud Director 10.2 since it needs some APIs implemented in VCD 10.3.
